### PR TITLE
Update for namespace tuples

### PR DESCRIPTION
**Pull request by RichLogan**
Update libquicr and add initial support for namespace tuples.

### CODE REVIEW

File: Decimus/Lib/libquicr/QCommon.h [hunk 1]
```diff
@@ -36,8 +36,10 @@ typedef struct QObjectHeaders {
 
 #ifdef __cplusplus
 static quicr::FullTrackName ftnConvert(QFullTrackName qFtn) {
+    const auto nameSpace = std::vector<std::uint8_t>(qFtn.nameSpace, qFtn.nameSpace + qFtn.nameSpaceLength);
+    std::vector<std::vector<std::uint8_t>> tuple = { nameSpace };
     return {
-        .name_space = std::vector<std::uint8_t>(qFtn.nameSpace, qFtn.nameSpace + qFtn.nameSpaceLength),
+        .name_space = quicr::TrackNamespace(tuple),
```
Comment (GhostofCookie) [inline Decimus/Lib/libquicr/QCommon.h:42]: Don't need to store this, could just do it inline

Comment (GhostofCookie) [inline Decimus/Lib/libquicr/QCommon.h:42]:
```suggestion
    return {
        .name_space = quicr::TrackNamespace(nameSpace),

Comment (GhostofCookie) [inline Decimus/Lib/libquicr/QCommon.h:42]: Or even shorter
```suggestion
    return {
        .name_space = quicr::TrackNamespace(quicr::Bytes(qFtn.nameSpace, qFtn.nameSpace + qFtn.nameSpaceLength)),

Comment (GhostofCookie) [reply Decimus/Lib/libquicr/QCommon.h:42]: This is actually a copy, so probably discourage this since copying the vector isn't needed.

**Comment by RichLogan** [comment]
Superseded by #516

**Pull request closed without merging**